service/acm/deserializers.go

```diff
@@ -2261,15 +2261,14 @@ func awsAwsjson11_deserializeDocumentAccessDeniedException(v **types.AccessDenie
 
 	for key, value := range shape {
 		switch key {
-		case "Message":
+		case "message", "Message":
 			if value != nil {
 				jtv, ok := value.(string)
 				if !ok {
 					return fmt.Errorf("expected ServiceErrorMessage to be of type string, got %T instead", value)
 				}
 				sv.Message = ptr.String(jtv)
 			}
-
```

Comment:
The changes to codegen here dropped this newline between cases across the board - it's making the diff harder to read, we should put it back.